docs/organization/dynamic-sampling/index.mdx

```diff
@@ -0,0 +1,219 @@
+
+<Alert>
+  Dynamic Sampling is available for customers on our latest Enterprise plans,
+  starting June, 2024.
+</Alert>
+
+## Prerequisites
+
+- [Admin-level permissions](/organization/membership/).
+- Latest version of one of the below SDKs:
+  - Python: 1.7.2 or later
+  - JavaScript: 7.6.0 or later
+  - Apple: 7.23.0 or later
+  - Android: 6.5.0 or later
+  - React Native: 4.3.0 or later
+  - Dart and Flutter: 6.11.0 or later
+  - PHP: 3.9.0 or later
+  - Laravel: 3.0.0 or later
+  - Symfony: 4.4.0 or later
+  - Ruby: 5.5.0 or later
+  - Java: 6.5.0 or later
+  - .NET: 3.22.0 or later
+  - Go: 0.16.0 or later
+
+## Configuration
+
+In this section you'll learn how to use Dynamic Sampling in your organization.
+
+![The dynamic sampling settings overview.](./img/mode-and-target-sample-rate.png)
+
+### Target Sample Rate
+
+To use Dynamic Sampling you have to specify a target sample rate for your entire organization, which determines how much incoming traffic should be stored across all your projects. Dynamic Sampling then distributes this sample rate across your projects automatically. Sample rates are adjusted continuously to account for changes in event volume over time.
+
+**The following adjustments are performed:**
+
+- Low-Volume Projects get a higher sample rate, ensuring important data isn’t missed.
+- High-Volume Projects have reduced sample rates to balance overall storage.
+- Sample rates are continuously adjusted based on recent traffic patterns, using a short sliding window to adapt to seasonal changes like holidays, weekends, and nighttime traffic.
+
+Dynamic Sampling also prioritizes the most valuable data within each project's allocated sample rate. You can learn more about this in the [Dynamic Sampling Priorities](#dynamic-sampling-priorities) section. This dynamic system ensures your sampling adapts to changing traffic and maintains balance across projects.
+
+**Project Preview**
+
+The [project preview](https://sentry.io/orgredirect/organizations/:orgslug/settings/dynamic-sampling/) table on the Dynamic Sampling settings page shows you how your project might be affected by the sample rate you've chosen. The preview shows the dynamically allocated rates for your projects based on either the last 24 hours or the last 30 days of traffic.
+
+![The dynamic sampling mode and target sample rate setting.](./img/project-sample-rate-preview.png)
+
+Because project rates continuously change, we recommend that you choose the 24-hour preview to get a more accurate view of your current sample rates for real-time analysis. We recommend the 30-day preview for seeing average sample rates in your billing cycle for long-term planning.
+
+<Alert>
+  If you deactivate advanced mode, the recalibration process will
+  start immediatly and the per-project sample rates will be reset to the
+  calculated values. There's no way to restore the manually configured project
+  sample rates once you've deactivated advanced mode.
+</Alert>
```

Comment:
I would move this into the "Advanced Mode" section below a paragraph that describes that you can deactivate advanced mode and go back to configuring the organization-wide target rate.